docs/design/autoware-interfaces/components/localization.md

```diff
@@ -0,0 +1,77 @@
+# Localization
+
+![Node diagram](images/Localization-Bus-ODD-Architecture.drawio.svg)
+
+## Inputs
+
+### PointCloud Map
+
+Environment map created with point cloud, Published by the Map Server.
+
+- sensor_msgs::msg::PointCloud2
+
+### Initial Pose
+
+Start pose of ego, Published by the User Interface or GNSS.
+
+- geometry_msgs::msg::PoseStamped
```

Comment:
It has been changed to a service `autoware_adapi_v1_msgs::srv::InitializeLocalization` to get result. Could you change to this? If necessary, you can define own messages for localization.
https://github.com/autowarefoundation/autoware_adapi_msgs

Reply:
thanks for your reply, I will change it soon.

Reply:
@isamu-takagi  hello:
After checking the latest code, I found my current tutorial is also acceptable, `rviz` published `initialpose` message is the `PoseWithCovarianceStamped` data type, autoware `ad_api_adaptors` package change it as a service to match the input of  the `pose_initializer` package.

Reply:
Yes, that's right. More precisely, It's an input to the API, not localization, but I'll leave it up to you to use `PoseWithCovarianceStamped` for simplicity.
https://autowarefoundation.github.io/autoware.universe/main/system/default_ad_api/document/localization/